package.json

```diff
@@ -1,91 +1,99 @@
-    "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json",
-    "prisma:generate": "prisma generate"
-  },
-  "dependencies": {
-    "@nestjs/common": "9.0.0",
-    "@nestjs/config": "2.3.3",
-    "@nestjs/core": "9.0.0",
-    "@nestjs/jwt": "10.0.3",
-    "@nestjs/passport": "9.0.3",
-    "@nestjs/platform-express": "9.0.0",
-    "@nestjs/typeorm": "9.0.1",
-    "@prisma/client": "4.15.0",
-    "bcrypt": "5.1.0",
-    "class-transformer": "0.5.1",
-    "class-validator": "0.14.0",
-    "dotenv": "16.1.4",
-    "passport": "0.6.0",
-    "passport-jwt": "4.0.1",
-    "passport-local": "1.0.0",
-    "pg": "8.11.0",
-    "reflect-metadata": "0.1.13",
-    "rxjs": "7.2.0",
-    "typeorm": "0.3.16"
-  },
-  "devDependencies": {
-    "@nestjs/cli": "9.0.0",
-    "@nestjs/schematics": "9.0.0",
-    "@nestjs/testing": "9.0.0",
-    "@types/bcrypt": "5.0.0",
-    "@types/express": "4.17.13",
-    "@types/jest": "29.5.1",
-    "@types/node": "18.16.12",
-    "@types/passport-jwt": "3.0.8",
-    "@types/passport-local": "1.0.35",
-    "@types/supertest": "2.0.11",
-    "@typescript-eslint/eslint-plugin": "5.0.0",
-    "@typescript-eslint/parser": "5.0.0",
-    "eslint": "8.0.1",
-    "eslint-config-prettier": "8.3.0",
-    "eslint-plugin-prettier": "4.0.0",
-    "jest": "29.5.0",
-    "prettier": "2.3.2",
-    "prisma": "4.15.0",
-    "source-map-support": "0.5.20",
-    "supertest": "6.1.3",
-    "ts-jest": "29.1.0",
-    "ts-loader": "9.2.3",
-    "ts-node": "10.9.1",
-    "tsconfig-paths": "4.2.0",
-    "typescript": "4.9.5"
-  },
-  "jest": {
-    "moduleFileExtensions": [
-      "js",
-      "json",
-      "ts"
-    ],
-    "rootDir": "src",
-    "testRegex": ".*\\.spec\\.ts$",
-    "transform": {
-      "^.+\\.(t|j)s$": "ts-jest"
+    "name": "recipe-app",
+    "version": "0.0.1",
+    "description": "",
+    "author": "",
+    "private": true,
+    "license": "UNLICENSED",
+    "scripts": {
+        "build": "nest build",
+        "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
+        "start": "nest start",
+        "start:dev": "nest start --watch",
+        "start:debug": "nest start --debug --watch",
+        "start:prod": "node dist/main",
+        "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
+        "test": "jest",
+        "test:watch": "jest --watch",
+        "test:cov": "jest --coverage",
+        "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
+        "test:e2e": "jest --config ./test/jest-e2e.json",
+        "prisma:generate": "prisma generate"
     },
-    "collectCoverageFrom": [
-      "**/*.(t|j)s"
-    ],
-    "coverageDirectory": "../coverage",
-    "testEnvironment": "node"
-  },
-  "prisma": {
-    "schema": "prisma/schema.prisma"
-  }
+    "dependencies": {
+        "@faker-js/faker": "8.0.2",
```

Comment:
you can move `@faker-js/faker` to `devDependencies - you don't need this on production

Reply:
hint: you can install libraries to `devDependencies` by using `--save-dev` or `-D`, e.g. `npm i @faker-js/faker -D`